lib/logstash/inputs/beats.rb

```diff
@@ -104,9 +104,9 @@ class LogStash::Inputs::Beats < LogStash::Inputs::Base
 
   # Enables storing client certificate information in event's metadata. You need 
   # to configure the `ssl_verify_mode` to `peer` or `force_peer` to enable this.
-  config :ssl_peer_metadata, :validate => :boolean, :default => false
+  config :ssl_peer_metadata, :validate => :boolean, :default => false, :deprecated => 'This option will be removed in the future, please add `ssl_peer_metadata` to `enrich` config to use.'
 
-  config :include_codec_tag, :validate => :boolean, :default => true
+  config :include_codec_tag, :validate => :boolean, :default => true, :deprecated => 'This option will be removed in the future, please add `include_codec_tag` to `enrich` config to use'
```

Comment:
When a deprecated option is used, the string provided here is an "extra" in a log message that already includes boiler-plate about deprecations and future removal, so we can be more clear with fewer words. The following would encourage our users to look for the docs for how to use the `enrich` option instead of giving them a hint that may have unexpected side-effects:

~~~ ruby
:deprecated => "use `enrich` option to configure which enrichments to perform"
~~~